gcp/api/fixtures/IntegrationTests_api_query_response.txt

```diff
@@ -1,120 +1,244 @@
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"name": "crossbeam-utils", "purl": "pkg:pypi/numpy@8.24.0"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"name": "crossbeam-utils", "version": "0.8.5"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"name": "crossbeam-utils"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"name": "numpy", "purl": "pkg:cargo/crossbeam-utils", "version": "0.8.5"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"name": "numpy", "purl": "pkg:cargo/crossbeam-utils"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"name": "numpy", "purl": "pkg:cargo/crossbeam-utils@0.8.5", "version": "0.8.5"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"name": "numpy", "purl": "pkg:cargo/crossbeam-utils@0.8.5"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"name": "numpy", "purl": "pkg:pypi/numpy", "version": "0.8.5"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"name": "numpy", "purl": "pkg:pypi/numpy"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"name": "numpy", "purl": "pkg:pypi/numpy@8.24.0", "version": "0.8.5"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"name": "numpy", "purl": "pkg:pypi/numpy@8.24.0"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"name": "numpy", "version": "0.8.5"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"name": "numpy"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"purl": "pkg:cargo/crossbeam-utils", "version": "0.8.5"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"purl": "pkg:cargo/crossbeam-utils"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"purl": "pkg:cargo/crossbeam-utils@0.8.5", "version": "0.8.5"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"purl": "pkg:cargo/crossbeam-utils@0.8.5"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"purl": "pkg:pypi/numpy", "version": "0.8.5"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"purl": "pkg:pypi/numpy"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"purl": "pkg:pypi/numpy@8.24.0", "version": "0.8.5"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"purl": "pkg:pypi/numpy@8.24.0"}}\n',
-    '200:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": {"version": "0.8.5"}}\n',
+[   '400:{"commit": "d374094d8c49b6b7d288f307e11217ec5a502391", "package": '
```

Comment:
This long test has been skipped and hasn't been updated in a long time (it is also failing for the master branch). The script is generated using the command `TESTS_GENERATE=1`. Most of the 400 failures are caused by querying both the `PURL` and `package` name at the same time, which is invalid.

We could also re-enable long tests on the pipeline (run after merge), it doesn't run too long (maybe 1 min)